modules/serverless-logic-install-kn-workflow-binary-file.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-binary-file_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the binary file
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have installed Podman on your local machine. 
+
+.Procedure
+
+. Download the latest binary file, suitable for your environment, from the link:https://mirror.openshift.com/pub/cgw/serverless-logic/latest/[Serverless Logic download mirror] page.
+
+. Extract the selected Knative Workflow plugin binary file by running the following command:
++
+[source,terminal]
+----
+$ tar xvzf kn-workflow-linux-amd64.tar.gz
+----
+
+. Rename the Knative Workflow plugin binary file to `kn-workflow` by running the following command:
++
+[source,terminal]
+----
+$ mv kn kn-workflow
+----
+
+. Install the `kn-workflow` command as a plugin of the Knative CLI by running the following commands:
+
+.. Copy the `kn-workflow` binary file to a directory in your PATH, such as `/usr/local/bin`, and ensure the file name is `kn-workflow`:
++
+[source,terminal]
+----
+$ cp path/to/downloaded/kn-workflow /usr/local/bin/kn-workflow
+----
+
+.. Make the binary file executable:
++
+[source,terminal]
+----
+$ chmod +x /usr/local/bin/kn-workflow
+----
+
+.Verification
+. Run the following command to verify that the `kn-workflow` plugin is installed successfully:
++
+[source,terminal]
+----
+$ kn plugin list
+----
+
+. After installing the plugin, you can use `kn-workflow` to run the related subcommands.
++
+.Aliases to use workflow subcommand
+[source,terminal]
+----
+kn-workflow
+----
++
+.Example output of `kn-workflow help` command
+[source,terminal]
+----
+Manage OpenShift Serverless Logic Workflow projects
+
+Usage:
+  kn workflow [command]
+
+Aliases:
+  kn workflow, kn-workflow
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  create      Creates a new OpenShift Serverless Logic Workflow project
+  deploy      Deploy an OpenShift Serverless Logic Workflow project on Kubernetes via SonataFlow Operator
+  help        Help about any command
+  quarkus     Manage OpenShift Serverless Logic Workflow projects built in Quarkus
+  run         Run an OpenShift Serverless Logic Workflow project in development mode
+  undeploy    Undeploy an OpenShift Serverless Logic Workflow project on Kubernetes via SonataFlow Operator
+  version     Show the version
```

Comment:
this output looks outdated

Reply:
Do you have the latest one that I can include?